### ISSUE TITLE
OVH and 503 error

### ISSUE THREAD

**Issue by bansan85** [i-net-software/dokuwiki-plugin-multiorphan#39]
Just for the record.

My Dokuwiki is hosted in France by OVH and the community hosting.

The plug-in is working well but everytime I run it, I'm blacklisted for a short time (about 15 minutes). During this time, I have a 503 error. I think the plug-in is using to much cpu load.

It's not a big wiki (about 300 pages and 600 media).

I think this problem should be write some where in the readme.

**Comment by gamma** [comment]
Do you use the plugins export page which triggers each page individually or do you use one of the URL that fetches the requested pages all at once?

Would a slow down option help your problem?

**Comment by bansan85** [comment]
By "community hosting", I mean that there is multiple websites on the same server that share the same resources.

I used the button "Start" at the URL : ```doku.php?id=start&do=admin&page=multiorphan``` so I scanned the whole wiki.

I have no idea what the criteria for being blocked. One people asked and it looks that 5 URL requested by second is too much. But I don't think that MultiOrphans do GET Url. It directly reads files on the server. Right ?

Thinks for you help,

![MultiOrphans-parameters](https://user-images.githubusercontent.com/9248729/62972469-5cf9aa00-be14-11e9-8c76-1e53c9c9d8cd.png)
